data/drak missions.txt

```diff
@@ -36,6 +36,7 @@ mission "Drak guarding Sayaiban"
 		system Sayaiban
 		ship "Archon" "Watchful Gravetender"
 	on enter Sayaiban
+		log "Explored an uninhabited system deep in Korath space, and encountered an Archon who spoke inside your head. Seems this Sayaiban system is home to a 'swarm intelligence' the Korath created. Best steer clear of here."
```

Comment:
Honestly I'm surprised I didn't type more English spelling words. Must've got lucky. I had no idea that was the American English rule on quotations though!